### PR TITLE
Created an endpoint for tracking all Icelandic torrent sites, list maintained by kjarni.cc

### PR DESCRIPTION
**Pull request by HermannBjorgvin**
Tracks up-status, sll, domain, inviteOnly.

### CODE REVIEW

**Comment by HermannBjorgvin** [comment]
Made one commit and then 4 more to clean up my mistakes

**Comment by HermannBjorgvin** [comment]
Travis looks like he isn't complaining about me

**Comment by HermannBjorgvin** [comment]
Don't merge this yet, apparently its running on a laptop, and a random port

File: endpoints/tv/index.js [hunk 1]
```diff
@@ -18,7 +18,18 @@ app.get('/tv', function (req, res) {
             stod2bio: '/tv/stod2bio',
             stod3: '/tv/stod3',
             skjar1: '/tv/skjar1'
-          }
+          },
+          channels: [
```
Comment (koddsson) [inline endpoints/tv/index.js]: Why is this here? Looks unrelated.

File: endpoints/torrents/tests/integration_test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+var request = require('request');
+var assert = require('assert');
+var helpers = require('../../../lib/test_helpers.js');
+
+describe('torrents', function() {
+    // I just copied the cinema test so i hope this is correct
```
Comment (koddsson) [inline endpoints/torrents/tests/integration_test.js]: I'd remove lines 6-15. The test looks good to me.

File: endpoints/tv/index.js (hunk 1 above)
Comment (HermannBjorgvin) [inline endpoints/tv/index.js]: It's unrelated, added it by accident. Don't know git well enough to undo it.

File: endpoints/torrents/tests/integration_test.js (hunk 2 above)
Comment (HermannBjorgvin) [inline endpoints/torrents/tests/integration_test.js]: Yes, good point

**Comment by kristjanmik** [comment]
Closed due to inactivity. Please create a new pull request if you have some changes.
